poc_mobile_verifier/src/verifier.rs

```diff
@@ -150,6 +127,45 @@ impl VerifierDaemon {
 
         Ok(())
     }
+
+    async fn last_verified_end_time(&self) -> Result<DateTime<Utc>> {
+        Ok(Utc.timestamp(meta::get(&self.pool, "last_verified_end_time").await?, 0))
+    }
+
+    async fn save_last_verified_end_time(
+        exec: impl PgExecutor<'_>,
```

Comment:
so the transaction also implements the PgExecutor? it makes sense to use the transaction in the context you've got it but independent of that this function signature seems strange that the getter uses `&self.pool` but the setter expects you to pass it a PgExecutor; is there a more specific type/trait you can gate this on or at least comment why this is set up the way it is to avoid future confusion?

Reply:
agreed, I didn't like it when I wrote it.  I think the function should just take the PgExecutor because it doesn't need to know whether it is in a transaction or not.  I could change the reader functions to also accept exec parameter to make them all the same, or I could create an `Option` in the struct for the transaction and use that if available or pool if not, but I like that the least.

Reply:
it's not terrible as-is; if you do anything i'd just make the getter's take an executor as well but it's probably fine already